packages/pytest-simcore/src/pytest_simcore/playwright_utils.py

```diff
@@ -1,14 +1,109 @@
 import json
-from dataclasses import dataclass
+import logging
+from collections.abc import Iterator
+from contextlib import ExitStack, contextmanager
+from dataclasses import dataclass, field
 from enum import Enum, unique
-from typing import Any, Final
+from types import SimpleNamespace
+from typing import Any, Final, TypeAlias
 
 from playwright.sync_api import WebSocket
 
 SECOND: Final[int] = 1000
 MINUTE: Final[int] = 60 * SECOND
 
 
+class DynamicIndentFormatter(logging.Formatter):
+    indent_char = "\t"
```

Comment:
This one looks like a constant so it should be upper case

Reply:
This is not a constant but a class attributes with a default value . I will annotate it